src/desktop/components/inquiry_questionnaire/views/inquiry.coffee

```diff
@@ -8,10 +8,12 @@ alertable = require '../../alertable_input/index.coffee'
 hasSeen = require '../../has_seen/index.coffee'
 sd = require('sharify').data
 template = -> require('../templates/inquiry.jade') arguments...
+{ repcaptcha } = require "@artsy/reaction/dist/Utils/repcaptcha"
```

Comment:
#sosominor is there a typo in reCaptcha (it's rep-). funny!

Reply:
this was actually driving me insane while working on it, gonna fix it on the reaction side!